portfolio/src/main/webapp/style.css

```
@@ -16,6 +19,34 @@
   flex-direction: row;
   justify-content: space-between;
 }
+.flex-col {
```

[4.1.4](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters): Separate words in class names by a hyphen.
```suggestion
.flex-col {
```

Here and elsewhere.